lib/SampleService/core/validator/unit_definitions.txt

```diff
@@ -0,0 +1 @@
+cell = count = cl
```

Comment:
Do you really want to define it as a count? If so it's convertible to anything else that's a count if I understand pint correctly

Reply:
I'm not sure what the alternative is here

Reply:
`cell = []`

That, IIUC, defines a new dimension. Units are converted along dimensions so `cell` shouldn't be converted to anything else.

I'm not a `pint` expert though. That's based on looking through the docs and source code last night

Reply:
Updated this to `cell = [] = cl`

Reply:
Are you sure `cl` doesn't clash with any other units?

Reply:
now that I think about it, it may clash with `centiliters`. Not a very common unit but there probably is no need to have an abbreviation here.

Reply:
I've removed the abbreviation